fw-child/resources/js/child-functions.js

```diff
@@ -609,47 +609,77 @@ const $ = jQuery;
     
       $('.scroll-card').each(function () {
         const card = this;
-    
+        const isLastCard = $(card).closest('.fw-query-item').is(':last-child');
+
+         // Set pointer-events to none at the beginning for every card
+        $(card).css('pointer-events', 'none');
+        
+        // Define common properties
+        let fromToProps = { 
+          y: '-85%',
+          opacity: 0,
+          scale: 0.75,
+          zIndex: 0
+        };
+        let toProps1 = {
+          y: '-15%',
+          opacity: 1,
+          scale: 1,
+          zIndex: 100,
+          onStart: function() {
+            $(card).css('pointer-events', ''); // Enable pointer-events as soon as the animation starts
+          },
+          onReverseComplete: function() {
+            $(card).css('pointer-events', 'none'); // Disable pointer-events when reversing the animation
+          }
+        };
+        let toProps2 = {
+          y: '55%',
+          opacity: 0,
+          scale: 0.75,
+          zIndex: 0,
+          ease: 'power2.in',
+          onComplete: function() {
+            $(card).css('pointer-events', 'none');  // Disable pointer-events when this animation is complete
+          },
+          onReverseComplete: function() {
+            $(card).css('pointer-events', '');  // Enable pointer-events when this animation is complete on reverse
+          }
+        };
+        
+        // Customize properties for the last card
+        if (isLastCard) {
+          toProps2.y = '0%';
+          toProps2.opacity = 1;
+          toProps2.scale = 1;
+          toProps2.zIndex = 100;
+        }
+      
         const timeline = gsap.timeline({
           scrollTrigger: {
             trigger: card,
             start: 'top 100%',
             end: 'top -50%',
             scrub: true,
-            markers: false    // Set to true to debug
+            markers: false,    // Set to true to debug
+            onLeave: function() {
+              $(card).css('pointer-events', 'none');  // Disable pointer-events when card leaves
+            },
+            onEnterBack: function() {
+              $(card).css('pointer-events', '');  // Disable pointer-events when card enters back
+            }
           }
         });
-    
+      
         timeline.fromTo(card, 
-          { 
-            y: '-85%',
-            opacity: 0,
-            scale: 0.75,
-            zIndex: 0
-          },
-          {
-            y: '-85%',
-            opacity: 0,
-            scale: 0.75,
-            zIndex: 0
-          }
+          fromToProps,
```

Comment:
The lines 674, 678 and 681 could now be on single lines since their second parameter has been replaced by a variable. I would also make it clearer that the functions are all called on the `timeline` object.

Example:
```javascript
timeline
    .fromTo(card, fromToProps, fromToProps)
    .to(card, toProps1)
    .to(card, toProps2);
```